service/src/test/java/bio/terra/workspace/app/configuration/external/controller/FolderApiControllerTest.java

```diff
@@ -385,29 +376,137 @@ public void updateFolders_parentFolderNotExists_throws404() throws Exception {
         HttpStatus.SC_NOT_FOUND);
   }
 
-  private ApiFolder createFolder(UUID workspaceId, String displayName) throws Exception {
-    return createFolder(workspaceId, displayName, /*description=*/ null, /*parentFolderId=*/ null);
+  @Test
+  public void updateFolderProperties_updateSuccessfully() throws Exception {
```

Comment:
There should be a test that calling updateFolderProperties leaves existing properties alone. Also should test "adding property" vs "changing existing property". I recommend changing this test.

```
// Create folder with property foo=bar

// Add property cake=lava

// Assert both properties

// Update cake=lava to cake=storm

// Assert both properties
```

Also